src/EditorFeatures/CSharpTest/Diagnostics/Suppression/SuppressionTest_FixAllTests.cs

```diff
@@ -0,0 +1,657 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Editor.UnitTests.Diagnostics;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Diagnostics.Suppression
+{
+    public abstract partial class CSharpSuppressionTests : AbstractSuppressionDiagnosticTest
+    {
+        #region "Fix all occurrences tests"
+
+        #region "Pragma disable tests"
+
+        public abstract partial class CSharpPragmaWarningDisableSuppressionTests : CSharpSuppressionTests
+        {
+            public partial class UserInfoDiagnosticSuppressionTests : CSharpPragmaWarningDisableSuppressionTests
```

Comment:
I retained the existing code structure - mainly aimed at defining more subtypes, but could be organized as top level subtypes rather then multiply nested types.